generator/test-primatives/template.templ

```diff
@@ -0,0 +1,22 @@
+package testprimatives
+
+templ render() {
+	<ul>
+		<li>{ int(1) }</li>
+		<li>{ int8(1) }</li>
+		<li>{ int16(1) }</li>
+		<li>{ int32(1) }</li>
+		<li>{ int64(1) }</li>
+		<li>{ uint(1) }</li>
+		<li>{ uint8(1) }</li>
+		<li>{ uint16(1) }</li>
+		<li>{ uint32(1) }</li>
+		<li>{ uint64(1) }</li>
+		<li>{ float32(1) }</li>
+		<li>{ float64(1) }</li>
+		<li>{ true }</li>
+		<li>{ false }</li>
+		<li>{ complex64(10 + 11i) }</li>
+		<li>{ complex128(10 + 11i) }</li>
+	</ul>
```

Comment:
Feel like we should have some strings, and some string-ish things alongside, just for completeness.